docker/scripts/install_common_deps.sh

```diff
@@ -17,16 +17,17 @@ sudo apt-get install --no-install-recommends -y \
   cppcheck \
   curl \
   git \
-  g++-8 \
-  pkg-config \
+  g++11 \
```

Comment:
We can just install the default version
```suggestion
  g++ \
```